content/try-opensds/installation.md

```diff
@@ -12,7 +12,7 @@ SODA Foundation Projects may be installed, deployed and tested using multiple to
 
 The recommended method of installation is using ansible in Ubuntu 16.04. Detailed steps are provided below.
 
-* [Local Cluster installation using ansible](https://github.com/sodafoundation/api/wiki/SODA-Projects-Cluster-Installation-through-Ansible)
+* [Local Cluster installation using ansible](installation-using-ansible.md)
```

Comment:
This link doesn't work.  
I think you need to change it to
`[Local Cluster installation using ansible](try-opensds/installation-using-ansible)`
Note the missing `.md`.

Reply:
I have added link to new installation readme. I have checked and link works in my code editor.